src/panels/config/integrations/integration-panels/mqtt/mqtt-config-panel.ts

```diff
@@ -33,6 +33,9 @@ class HaPanelDevMqtt extends LitElement {
   @LocalStorage("panel-dev-mqtt-retain-ls", true, false)
   private retain = false;
 
+  @LocalStorage("panel-dev-mqtt-allow-template-ls", true, false)
+  private allow_template = false;
```

Comment:
```suggestion
  private _allowTemplate = false;
```

Reply:
Let's give the other local vars also the `_` prefix in a follow-up PR.

Reply:
See: https://github.com/home-assistant/frontend/pull/14898 for a follow up on the other local vars.